cmd/create.go

```diff
@@ -40,39 +40,42 @@ func CreateCmd() *cobra.Command {
 	Once the API model has been created, a user can optionally change the model to their liking.
 	After a model is defined and configured properly, the user can then apply the model.`,
 		Run: func(cmd *cobra.Command, args []string) {
-			if len(args) == 0 {
-				co.Name = cli.StrEnvDef("KUBICORN_NAME", namer.RandomName())
-			} else if len(args) > 1 {
+			switch len(args) {
+			case 0:
+				co.Name = viper.GetString(keyKubicornName)
+				if co.Name == "" {
+					co.Name = namer.RandomName()
+				}
+			case 1:
+				co.Name = args[0]
+			default:
 				logger.Critical("Too many arguments.")
 				os.Exit(1)
-			} else {
-				co.Name = args[0]
 			}
 
-			err := RunCreate(co)
-			if err != nil {
+			if err := RunCreate(co); err != nil {
 				logger.Critical(err.Error())
 				os.Exit(1)
 			}
 
 		},
 	}
 
-	createCmd.Flags().StringVarP(&co.StateStore, "state-store", "s", cli.StrEnvDef("KUBICORN_STATE_STORE", "fs"), "The state store type to use for the cluster")
-	createCmd.Flags().StringVarP(&co.StateStorePath, "state-store-path", "S", cli.StrEnvDef("KUBICORN_STATE_STORE_PATH", "./_state"), "The state store path to use")
-	createCmd.Flags().StringVarP(&co.Profile, "profile", "p", cli.StrEnvDef("KUBICORN_PROFILE", "google"), "The cluster profile to use")
-	createCmd.Flags().StringVarP(&co.CloudID, "cloudid", "c", cli.StrEnvDef("KUBICORN_CLOUDID", ""), "The cloud id")
-	createCmd.Flags().StringVarP(&co.Set, "set", "e", cli.StrEnvDef("KUBICORN_SET", ""), "Override values in the default profile. EX: --set=ssh.publicKey=~/.ssh/my_key.pub")
+	fs := createCmd.Flags()
+
+	fs.StringVarP(&co.StateStore, keyStateStore, "s", viper.GetString(keyStateStore), descStateStore)
```

Comment:
nit: maybe we should consider factoring out the common code in most of these `cmd` files. but agreed with @xmudrii, doesnt have to be addressed in this PR.